src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -39,13 +39,16 @@ is dropped on a creadcrumb.
 				<Breadcrumb title="Folder 1" href="/Folder 1" />
 				<Breadcrumb title="Folder 2" href="/Folder 1/Folder 2" :disable-drop="true" />
 				<Breadcrumb title="Folder 3" href="/Folder 1/Folder 2/Folder 3" />
-				<Breadcrumb title="Folder 4" href="/Folder 1/Folder 2/Folder 3/Folder 4" />
+				<Breadcrumb title="Folder 4" href="/Folder 1/Folder 2/Folder 3/Folder 4">
 				<Breadcrumb title="Folder 5" href="/Folder 1/Folder 2/Folder 3/Folder 4/Folder 5" :disable-drop="true">
-					<Actions>
-						<ActionButton icon="icon-share" @click="alert('Share')">
-							Share
-						</ActionButton>
-					</Actions>
+					<ActionButton icon="icon-share" @click="alert('Share')">
+						Share
+					</ActionButton>
+				</Breadcrumb>
+				<Breadcrumb title="Folder 6" href="/Folder 1/Folder 2/Folder 3/Folder 4/Folder 5/Folder 6" :forceMenu="true">
```

Comment:
```suggestion
				<Breadcrumb title="Folder 6" href="/Folder 1/Folder 2/Folder 3/Folder 4/Folder 5/Folder 6">
```
Keep it simple I'd say :wink:

Reply:
Something is messed up anyway. I will check.

Reply:
Okay, I'll stop reviewing and let you clean things up! This is a phenomenal work!! :tada: 
Loved it a lot!!!

Ping me again when you need me! :)